### DISCUSSION TITLE
Light control: Color wheel and color temperature on same tab

### DISCUSSION THREAD

**Discussion by henning334** [home-assistant/frontend#10408] [Dashboard]
I've come over pictures of what i now know is an older version of HA / lovelace.

In the picture, the color wheel and color temperature slider are visible in the same page/tab/at the same time.

In my setup i instead have two buttons, "Color" and "Temperature" that i have to switch between.
This is an unecessary click every time i wanna control my lights, and the buttons take up as much room as the temperature slider itself, so there's no room saved either (not that that is any concern in the first place).

Thanks for the great job you do! Hope you'll change this (back) :)

**Comment by bramkragten** [comment]
Maybe you understand this choice better if you know we also support lights like these:
![image](https://user-images.githubusercontent.com/5662298/139341933-1a5bd75b-9aa3-49ae-9a20-3aa7d33ddc0c.png)

It is an RGBWW light, so it allows you to control the RGB and 2 white channels, but could also support color temperature.

Showing all of that on the same page gets really confusing, so we decided to separate the controls based on light modes.

We only show the controls that are relevant for the chosen light mode.

**Reply by henning334** [reply]
Well it kind of make sense in those scenarios. Could it be possible to show the temperature and color in one page if it is NOT a RGBWW-light? Or even better: make a setting so we can choose ourselves?
Thanks again for the great work you guys do on HA! 💯
